### ISSUE TITLE
NPE in gateway when gateway defines oneWayTls but virtualservice does not have an sslConfig

### ISSUE THREAD

**Issue by jack0** [solo-io/gloo#5262]
**Describe the bug**
If a Gateway CR is defined to enable oneWayTls, but a VirtualService does not define an sslConfig, the gateway will run into a nil pointer dereference error and crash.

**To Reproduce**
Configure a gateway to enable oneWayTls. Follow the https redirect example, the http VirtualService without sslConfig will cause the gateway deployment to crash.

**Expected behavior**
A VirtualService without an sslConfig should not cause gateway to crash.

**Additional context**
I am using Gloo v1.8.10 but the nil pointer deference occurs here: https://github.com/solo-io/gloo/blame/v1.8.11/projects/gateway/pkg/translator/http.go#L109

Code should be:

```golang
if vs.GetSslConfig() != nil && vs.GetSslConfig().GetOneWayTls() == nil {
  vs.GetSslConfig().OneWayTls = &wrappers.BoolValue{Value: val.GetValue()}
}
```

**Comment by kevin-shelaga** [comment]
@sam-heilbron Yes, will do.

**Comment by sam-heilbron** [comment]
This has been fixed in Gloo 1.8.x and 1.9.x. The fix has not been deployed yet, but will be included in upcoming releases. Thanks for the discovering this @jack0
